src/otx/algo/explain/explain_algo.py

```diff
@@ -359,7 +360,7 @@ def average_and_normalize(
         if isinstance(pred, dict):
             masks, scores, labels = pred["masks"], pred["scores"], pred["labels"]
         else:
-            masks, scores, labels = (pred.masks, pred.scores, pred.labels)
+            masks, scores, labels = (pred.masks, pred.scores, pred.labels)  # type: ignore[attr-defined]
```

Comment:
@negvet @GalyaZalesskaya Is the `pred` used here an `InstanceData`? I think it's a `**PredEntity`, could you check?